MapboxNavigation/Resources/fr.lproj/Abbreviations.csv.plist

```diff
@@ -88,145 +88,551 @@
 &quot;zone industrielle&quot;, &quot;ZI&quot;
 </string>
     <key>classifications</key>
-    <string>&quot;agricole&quot;, &quot;Agr.&quot;
+    <string>&quot;adjudant&quot;, &quot;Adj.&quot;
+&quot;administratif&quot;, &quot;Admin.&quot;
```

Comment:
Something looks off here.

Reply:
This is, unfortunately, the correct format until we figure out https://github.com/mapbox/mapbox-navigation-ios/issues/1050. French abbreviations loads correctly into the dictionary:
<details>
Optional(["classifications": ["cathédrale": "Cathéd.", "coopératif": "Coop.", "siècle": "Si.", "territoire": "Territ.", "étage": "ét.", "samedi": "sa.", "commune": "Commun.", "fluviale": "Flv", "églises": "Égl.", "région": "Rég.", "fondateur": "Fond.", "prince": "Pce", "majeures": "Maj.", "centrale": "Ctrale", "américains": "Amér.", "provence-alpes-côte d\'azur": "PACA", "associés": "Assoc.", "école": "Éc.", "communautés": "Commtés", "sociale": "Soc.", "avant": "av.", "conférence": "Confér.", "antenne": "Ant.", "boucher": "Bouch.", "basses": "Bas.", "commerce": "Commerc.", "présidents": "Pdts", "civil": "Civ.", "sportive": "Sport.", "petite": "Pet.", "européen": "Eur.", "xiiie": "13e", "hauts": "Hts", "vers": "v/", "rivière": "Riv.", "sur": "/", "lieutenant": "Lieut.", "musique": "Musiq.", "médical": "Méd.", "moyennes": "Moy.", "hautes": "Htes", "district": "Distr.", "départementaux": "Dépt", "animation": "Anim.", "médecin": "Méd.", "universel": "Univ.", "postale": "Post.", "forestier": "Forest.", "majeur": "Maj.", "apostolique": "Apost.", "naval": "Nav.", "barrage": "Barr.", "ixe": "9e", "pontifical": "Pontif.", "iiie": "3e", "agronomiques": "Agro.", "archéologique": "Archéo.", "ministère": "Min.", "xixe": "19e", "xiv": "14", "exploitation": "Exploit.", "administrations": "Admin.", "sapeur": "Sap.", "public": "Publ.", "oriental": "Ori.", "haute": "Hte", "international": "Int.", "xiie": "12e", "société anonyme": "SA", "petites": "Pet.", "centres": "Ctres", "xviie": "18e", "viticole": "Vitic.", "publique": "Publ.", "présidentes": "Pdtes", "lycée d\'enseignement professionnel": "LEP", "populaire": "Pop.", "archevêque": "Archev.", "animé": "Anim.", "secondaires": "Second.", "consulat": "Consul.", "septembre": "Sept.", "princesse": "Pse", "techniques": "Tech.", "mondiales": "Mond.", "colonel": "Col.", "aviatrice": "Aviat.", "infirmerie": "Infirm.", "protégés": "Prot.", "xxii": "22", "xie": "11e", "église": "Égl.", "communal": "Commun.", "développement": "Dévelop.", "dans": "d/", "xvii": "18", "industrielle": "Ind.", "centre hospitalier universitaire": "CHU", "viiie": "8e", "civiles": "Civ.", "commerçants": "Commerç.", "orientales": "Ori.", "municipalité": "Munic.", "saints": "Sts", "américaines": "Amér.", "iie": "2e", "sergent": "Sgt", "communale": "Commun.", "doyen": "Doy.", "ophtalmologues": "Ophtalmos", "médicale": "Méd.", "départementales": "Dépt", "directeur": "Dir.", "football": "Foot", "mademoiselle": "Mle", "premiers": "1ers", "pontificaux": "Pontif.", "vendredi": "ve.", "numéro": "Nº", "xve": "15e", "archives": "Arch.", "viticulture": "Vitic.", "primaires": "Prim.", "docteurs": "Drs", "régionale": "Rég.", "fédération": "Féd.", "tertiaires": "Terti.", "fluviales": "Flv", "agronome": "Agro.", "cafés": "Cfés", "démocratique": "Dém.", "nautique": "Naut.", "commerciale": "Commerc.", "national": "Nal", "trafic": "Traf.", "janvier": "Janv.", "charcutier": "Charc.", "hôtels": "Hôt.", "nationales": "Nales", "département": "Dépt", "central": "Ctral", "municipale": "Munic.", "occidental": "Occ.", "ive": "4e", "tertiaire": "Terti.", "congrégations": "Congrég.", "infanterie": "Infant.", "métros": "Mº", "directrices": "Dir.", "ferroviaire": "Ferr.", "pontificales": "Pontif.", "xvie": "17e", "professeur": "Prof.", "fleuve": "Flv", "spécial": "Spéc.", "asiatique": "Asiat.", "mosquée": "Mosq.", "mondiale": "Mond.", "deux mil": "2000", "provincial": "Prov.", "militaires": "Milit.", "cafétérias": "Cafét.", "monsieur": "M.", "union": "U.", "cliniques": "Clin.", "sapeur-pompiers": "Sap.-pomp.", "pharmacie": "Pharm.", "grands": "Gds", "sous-préfet": "Ss-préf.", "syndicat": "Synd.", "congrégation": "Congrég.", "pharmacien": "Pharm.", "catholique": "Cathol.", "lycée": "Lyc.", "millénaire": "Mill.", "boucherie": "Bouch.", "naturel": "Natur.", "sous": "s/s", "fondation": "Fond.", "jeudi": "je.", "technologique": "Techno.", "spéciales": "Spéc.", "métropolitaine": "Métrop.", "spectacle": "Spect.", "postal": "Post.", "centre d\'enseignement du second degré": "CES", "agriculteur": "Agric.", "européennes": "Eur.", "société anonyme à responsabilité limitée": "SARL", "africaine": "Afric.", "catholiques": "Cathol.", "xx": "20", "mardi": "ma.", "mineur": "Min.", "lundi": "lu.", "hélicoptère": "hélico", "bâtiment": "Bât.", "ix": "9", "départementale": "Dépt", "matin": "mat.", "municipaux": "Munic.", "ophtalmologue": "Ophtalmo", "occidentale": "Occ.", "ire": "1re", "traversière": "Traver.", "métropolitain": "Métrop.", "rural": "Rur.", "civile": "Civ.", "américaine": "Amér.", "associative": "Assoc.", "asiatiques": "Asiat.", "x": "10", "centrales": "Ctrales", "province": "Prov.", "urbanisme": "Urb.", "aviateurs": "Aviat.", "syndicats": "Synd.", "fluviaux": "Flv", "théâtrale": "Thé.", "lieutenant-colonel": "Lieut.-Col.", "après": "apr.", "mairie": "Mair.", "spécialisé": "Spéc.", "sapeur-pompier": "Sap.-pomp.", "croix": "Cx", "société nationale": "SN", "occidentaux": "Occ.", "européenne": "Eur.", "junior": "Jr", "doyens": "Doy.", "commerçante": "Commerç.", "administration": "Admin.", "routière": "Rout.", "cinéma": "Ciné", "musée": "Msée", "iii": "3", "universitaire": "Univ.", "secondaire": "Second.", "xxiii": "23", "viticulteurs": "Vitic.", "cimetière": "Cim.", "global": "Glob.", "xix": "19", "sainte": "Ste", "comte": "Cte", "administrative": "Admin.", "spéciaux": "Spéc.", "protection": "Prot.", "navale": "Nav.", "portuaire": "Port.", "agriculture": "Agric.", "aviatrices": "Aviat.", "infirmières": "Infirm.", "confédérale": "Conféd.", "fondateurs": "Fond.", "information": "Info.", "bibliothèque": "Biblio.", "pompiers": "Pomp.", "préfectorale": "Préf.", "xv": "15", "urbaine": "Urb.", "districts": "Distr.", "décembre": "Déc.", "commerçant": "Commerç.", "administratif": "Admin.", "général": "Gal", "camping": "Camp.", "octobre": "Oct.", "hôpital": "Hôp.", "monseigneur": "Mgr", "xii": "12", "iv": "4", "haut": "Ht", "immeuble": "Imm.", "établissements": "Étabts", "municipales": "Munic.", "fondatrices": "Fond.", "collège": "Coll.", "direction": "Dir.", "universelle": "Univ.", "spectacles": "Spect.", "son altesse": "S.Alt.", "jésus-christ": "J.-C.", "orientale": "Ori.", "amiral": "Amir.", "archiduc": "Archid.", "alpin": "Alp.", "sociétés": "Stés", "mercredi": "me.", "saint": "St", "protégé": "Prot.", "vii": "7", "enseignement": "Enseign.", "développé": "Dévelop.", "industriel": "Ind.", "poste": "Post.", "centre": "Ctre", "maternelle": "Matern.", "dimanche": "di.", "avril": "Avr.", "major": "Mjr", "restaurant": "Restau.", "piscine": "Pisc.", "républicain": "Républ.", "moyens": "Moy.", "amiraux": "Amir.", "maréchal": "Mal", "musicien": "Music.", "alpine": "Alp.", "navigable": "Navig.", "xxie": "21e", "v": "5", "société": "Sté", "privé": "Priv.", "campings": "Camp.", "maritime": "Marit.", "musical": "Music.", "chapelle": "Chap.", "grande": "Gde", "adjudant": "Adj.", "étang": "Étg", "présidente": "Pdte", "préfecture": "Préf.", "deuxième": "2e", "xe": "10e", "comtés": "Ctés", "généraux": "Gaux", "grandes": "Gdes", "viticulteur": "Vitic.", "culturel": "Cult.", "comté": "Cté", "européens": "Eur.", "protégée": "Prot.", "première": "1re", "présidentielle": "Pdtle", "monument": "Mmt", "cinémas": "Cinés", "banque": "Bq", "directions": "Dir.", "châteaux": "Chât.", "île-de-france": "IDF", "hôtel": "Hôt.", "vétérinaires": "Vét.", "écoles": "Éc.", "aviateur": "Aviat.", "palais": "Pal.", "informations": "Infos", "supérieure": "Sup.", "majeurs": "Maj.", "résidentielle": "Résid.", "sapeurs": "Sap.", "mineures": "Min.", "juillet": "Juil.", "commercial": "Commerc.", "préfet": "Préf.", "moyenne": "Moy.", "grand": "Gd", "agriculteurs": "Agric.", "vi": "6", "viticoles": "Vitic.", "maire": "Mair.", "nationale": "Nale", "urbaines": "Urb.", "ambassade": "Amb.", "millénaires": "Mill.", "mondial": "Mond.", "café": "Cfé", "docteur": "Dr", "terrain": "Terr.", "président": "Pdt", "forestière": "Forest.", "globale": "Glob.", "globaux": "Glob.", "musicale": "Music.", "africaines": "Afric.", "ve": "5e", "xive": "14e", "février": "Févr.", "temple": "Tple", "américain": "Amér.", "ier": "1er", "crédit": "Créd.", "technologie": "Techno.", "hospice": "Hosp.", "supérieur": "Sup.", "départemental": "Dépt", "mémorial": "Mémor.", "administratives": "Admin.", "territorial": "Territ.", "métro": "Mº", "république": "Rép.", "historiques": "Hist.", "territoriale": "Territ.", "arrêt": "Arr.", "ophtalmologie": "Ophtalm.", "ii": "2", "préfectoral": "Préf.", "mineurs": "Min.", "supérieures": "Sup.", "confédération": "Conféd.", "établissement": "Étabt", "alpins": "Alp.", "ferré": "Ferr.", "viie": "7e", "provinciale": "Prov.", "monuments": "Mmts", "médicales": "Méd.", "après-midi": "a.-m.", "archiduchesse": "Archid.", "théâtral": "Thé.", "pharmacienne": "Pharm.", "archéologie": "Archéo.", "zone": "zon.", "novembre": "Nov.", "vie": "6e", "confédéral": "Conféd.", "hospitalier": "Hosp.", "unité": "U.", "hôtels-restaurants": "Hôt.-Rest.", "clinique": "Clin.", "agronomes": "Agro.", "moyen": "Moy.", "professionnel": "Profess.", "catégorie": "Catég.", "régional": "Rég.", "mineure": "Min.", "ferrée": "Ferr.", "infirmier": "Infirm.", "xxe": "20e", "xvi": "17", "troisième": "3e", "infirmiers": "Infirm.", "orientaux": "Ori.", "technologiques": "Techno.", "aviation": "Aviat.", "conférences": "Confér.", "directeurs": "Dir.", "associé": "Assoc.", "pontificale": "Pontif.", "sous-préfecture": "Ss-préf.", "alpines": "Alp.", "exploitants": "Exploit.", "caporal": "Capo.", "madame": "Mme", "social": "Soc.", "sa majesté": "S.Maj.", "rurale": "Rur.", "commerciaux": "Commerc.", "commerces": "Commerc.", "hélicoptères": "hélicos", "service": "Sce", "navigation": "Navig.", "résidentiel": "Résid.", "communauté": "Commté", "pâtissier": "Pâtiss.", "musicienne": "Music.", "centre hospitalier régional": "CHR", "municipal": "Munic.", "scolaire": "Scol.", "infirmière": "Infirm.", "majeure": "Maj.", "hôtel-restaurant": "Hôt.-Rest.", "stade": "Std", "maritimes": "Marit.", "spécialisée": "Spéc.", "générale": "Gale", "petits": "Pet.", "senior": "Sr", "mondiaux": "Mond.", "électriques": "Électr.", "fédérale": "Féd.", "agricoles": "Agric.", "vétérinaire": "Vét.", "collèges": "Coll.", "républicaine": "Républ.", "commerciales": "Commerc.", "son altesse royale": "S.A.R.", "globales": "Glob.", "agronomique": "Agro.", "civils": "Civ.", "administratifs": "Admin.", "occidentales": "Occ.", "technologies": "Techno.", "cafétéria": "Cafét.", "parachutistes": "Para.", "chapelles": "Chap.", "protégées": "Prot.", "électrique": "Électr.", "privée": "Priv.", "africains": "Afric.", "urbains": "Urb.", "théâtre": "Thé.", "arrondissement": "Arrond.", "médicaux": "Méd.", "petit": "Pet.", "faculté": "Fac.", "fédéral": "Féd.", "charcuterie": "Charc.", "professionnelle": "Profess.", "technique": "Tech.", "directrice": "Dir.", "fluvial": "Flv", "pompier": "Pomp.", "postaux": "Post.", "archipel": "Archip.", "militaire": "Milit.", "culturelle": "Cult.", "notre-dame": "ND", "coopérative": "Coop.", "régiment": "Régim.", "exploitant": "Exploit.", "hospices": "Hosp.", "élém": "Élém.", "premier": "1er", "numéros": "Nºˢ", "état": "É.", "xxi": "21", "anonyme": "Anon.", "cimetières": "Cim.", "fondatrice": "Fond.", "xiii": "13", "associatif": "Assoc.", "association": "Assoc.", "urbain": "Urb.", "basse": "Bas.", "château": "Chât.", "parachutiste": "Para.", "xi": "11", "historique": "Hist.", "supérieurs": "Sup.", "spéciale": "Spéc.", "agricole": "Agric.", "université": "Univ.", "jardin": "Jard.", "agronomie": "Agro.", "états": "É.", "présidentiel": "Pdtl", "messieurs": "MM", "sportif": "Sport.", "africain": "Afric.", "primaire": "Prim.", "viii": "8", "internationale": "Int."], "directions": ["sud-sud-ouest": "SSO", "est": "E", "sud": "S", "nord-nord-est": "NNE", "nord-ouest": "NO", "sud-ouest": "SO", "ouest-sud-ouest": "OSO", "ouest": "O", "ouest-nord-ouest": "ONO", "est-nord-est": "ENE", "nord": "N", "nord-est": "NE", "nord-nord-ouest": "NNO", "est-sud-est": "ESE", "sud-est": "SE", "sud-sud-est": "SSE"], "abbreviations": ["aérodrome": "Aérod", "aéroport": "Aérop", "allée": "All"]])
</details>